documentation/src/main/asciidoc/user_guide/chapter-5-Integration_with_other_frameworks.adoc

```diff
@@ -1191,7 +1191,7 @@ You activate Spring's cache support using xml:
 
 ----
 
-somewhere in your application context. This enable the cache annotations in Spring. Alternatively, it can be done programmatically:
+somewhere in your application context. This enable the cache annotations in Spring. Alternatively, it can be done pragmatically:
```

Comment:
@tristantarrant Created JIRAs for making dependencies provided:
* JCache (I believe we should do it since it is possible to use them with both small and uber jars). @galderz Would you agree with me? https://issues.jboss.org/browse/ISPN-6295
* Cache Stores: https://issues.jboss.org/browse/ISPN-6296